docs/Using-Fleet/REST-API.md

```diff
@@ -5480,21 +5504,28 @@ load balancer timeout.
   ]
 }
 ```
+
 ---
 
 ## Schedule
 
-- [Get schedule](#get-schedule)
-- [Add query to schedule](#add-query-to-schedule)
-- [Edit query in schedule](#edit-query-in-schedule)
-- [Remove query from schedule](#remove-query-from-schedule)
+> The Schedule API endpoints are deprecated as of Fleet 4.XX. It is maintained for backwards compatibility. 
```

Comment:
Why are we saying `4.XX` instead of `4.25.0` or the specific version this is merged into?

Reply:
Left it open because I was not sure which release it will actually be going into.